assets/pages/new-contributions-landing/contributionsLandingActions.js

```diff
@@ -398,15 +418,25 @@ const recurringPaymentAuthorisationHandlers = {
   DirectDebit: recurringPaymentAuthorisationHandler,
 };
 
-const paymentAuthorisationHandlers: PaymentMatrix<(Dispatch<Action>, State, PaymentAuthorisation) => void> = {
+const paymentAuthorisationHandlers: PaymentMatrix<(
+  Dispatch<Action>,
+  State,
+  PaymentAuthorisation,
+  onComplete?: PaymentResult => void,
+) => void> = {
   ONE_OFF: {
     PayPal: () => {
       // Executing a one-off PayPal payment happens on the backend in the /paypal/rest/return
       // endpoint, after PayPal redirects the browser back to our site.
       logException('Paypal one-off has no authorisation handler');
     },
-    Stripe: (dispatch: Dispatch<Action>, state: State, paymentAuthorisation: PaymentAuthorisation): void => {
-      dispatch(executeStripeOneOffPayment(stripeChargeDataFromAuthorisation(paymentAuthorisation, state)));
+    Stripe: (
+      dispatch: Dispatch<Action>,
+      state: State,
+      paymentAuthorisation: PaymentAuthorisation,
+      onComplete?: Function,
+    ): void => {
+      dispatch(executeStripeOneOffPayment(stripeChargeDataFromAuthorisation(paymentAuthorisation, state), onComplete));
```

Comment:
I noticed we use dispatch for events mostly, but there are some commands like this one too.  Is it just a case of mix and match or are there any rules on this?